README.md

```diff
@@ -1 +1,261 @@
-# Review-Your-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
```

Comment:
You should've done this section of deleting the comments and instructions since it is better for read me file.

Reply:
- [ ] As requested by the previous reviewer, you should give a reasonable explanation of why you requested the changes. Here you want the student to remove unnecessary comments from the code so why don't you explain to the student that removing unnecessary comments from his/her code will make the clean to read? :blush: